content/quickstart-hpo-experiment.rst

```diff
@@ -84,32 +104,70 @@ More information on Polaris job submitting (nodes, walltime, queue, etc...) can
     .. code-block:: JSON
 
         [
-        
+
+          {
+            "name": "train_ml_data_dir",
```

Comment:
Agreed, but not aware of a way around it. These are defined in the .txt file by the IMPROVE framework, and the only way to override that is with CL arguments. Only way to pass CL arguments right now is in the hyperparams file... I think... I could be wrong on these things.

Possibly, the easiest fix is to make these pathing arguments environmental variables in the IMPROVE framework, maybe sourced from .sh as a secondary overview file to our normal .txt. Then anything like could be exported in cfg-1.sh or cfg-my-settings.sh. Although, this would require some changes to the cross-study framework.

Reply:
Maybe @j-woz has a solution?